.github/workflows/cypress_e2e_tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Cypress E2E Tests
+
+on:
+  pull_request:
+      branches: [ master ]
+      paths:
+        - 'web/client/**'
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Cypress
+        run: npm i cypress@^12.10
```

Comment:
Do we need this command here if the `package.json` contains the `cypress` dependency? Using the `package.json` is preferable to avoid conflicts in the future.

Reply:
Removed that and installing packages from package.json instead.